Mathlib/Algebra/Group/AddChar.lean

```diff
@@ -400,13 +416,20 @@ lemma div_apply' (ψ χ : AddChar A M) (a : A) : (ψ / χ) a = ψ a / χ a := by
 lemma sub_apply' (ψ χ : AddChar A M) (a : A) : (ψ - χ) a = ψ a / χ a := by
   rw [sub_apply, map_neg_eq_inv, div_eq_mul_inv]
 
+@[simp] lemma zsmul_apply (n : ℤ) (ψ : AddChar A M) (a : A) : (n • ψ) a = ψ a ^ n := by
+  cases n
+  · simp
+  · simp [-neg_apply, neg_apply']
```

Comment:
could make this a cute one-line proof if you want

Reply:
Well spotted